Extensions/Kayenta/Src/Tasks/createCanaryAnalysisRequestV0/task.json

```diff
@@ -0,0 +1,298 @@
+    {
+      "name": "configAccountName",
+      "type": "string",      
+      "label": "Configuration Account",
+      "required": "true",
+      "helpMarkDown": "Configuration account name to fetch canary configurations. Ex : google-account",
+      "groupName": "accountConfiguration",
+      "properties": {
+        "EditableOptions": "True"
+        }
+    },
+    {
+      "name": "canaryConfig",
+      "type": "pickList",      
+      "label": "Canary config",
+      "required": "true",
+      "helpMarkDown": "Choose the canary-config for the analysis.",
+      "groupName": "accountConfiguration",
+      "properties": {
+        "EditableOptions": "False"
+        }
+    },
+    {
+        "name": "marginalThreshold",
+        "type": "string",       
+        "label": "Marginal threshold",
+        "required": "false",
+        "defaultValue": "0",
+        "helpMarkDown": "Canary classification will return Marginal if canary-score falls above marginal threshold else return Fail, Ex : 75.0",
+        "groupName": "thresholds",
+        "properties": {
+            "EditableOptions": "True"
+          }
+    },
+    {
+        "name": "passThreshold",
+        "type": "string",      
+        "label": "Pass threshold",
+        "required": "true",
+        "helpMarkDown": "Canary will Pass only if canary-score crosses pass threshold, Ex : 95.0",
+        "groupName": "thresholds",
+        "properties": {
+            "EditableOptions": "True"
+          }
+    },
+    {
+        "name": "lifetimeDurationMins",
+        "type": "string",      
+        "label": "Analysis lifetime duration {mins}",
+        "required": "true",
+        "helpMarkDown": "Analysis lifetime will continue for this duration, Ex : 45",
+        "groupName": "executionRequest",
+        "properties": {
+            "EditableOptions": "True"
+          }
+    },
+    {
+        "name": "lookbackMins",
+        "type": "string",       
+        "label": "Lookback {mins}",
+        "defaultValue": "0",
+        "required": "false",
+        "helpMarkDown":"If this value is supplied, judgements will be performed on a sliding time window:endTime - lookbackMins to startTime + (judgementNumber * interval). If this field is omitted, the judgements will be performed on a growing time window:startTime + (judgementNumber - 1 * interval) to startTime + (judgementNumber * interval)", 
+        "groupName": "executionRequest",
+        "properties": {
+            "EditableOptions": "True"
+          }
+    },
+    {
+        "name": "analysisIntervalMins",
+        "type": "string",    
+        "label": "Analysis interval {mins}",
+        "required": "true",
+        "helpMarkDown": "Judgement Analysis will occur after each interval during the lifetime of Analysis, Ex : 15",
+        "groupName": "executionRequest",
+        "properties": {
+            "EditableOptions": "True"
+          }
+    },
+    {
+      "name": "experimentScope",
+      "type": "string",       
```

Comment:
Keep Label and Name similar. Like label is "Canary scope" then use Name as "canaryScope". It is then easy to determine and read which input is mapped to which label. 
Please change this for all your inputs where you have differences.